buildkite/src/Lib/Cmds.dhall

```diff
@@ -51,9 +51,12 @@ let module = \(environment : List Text) ->
 
   let CompoundCmd = {
     Type = {
-      preprocess : Cmd,
-      postprocess : Cmd,
-      inner : Cmd
+      -- unpackage data downloaded from gcloud
```

Comment:
Yea, the fact that there's this expectation on a "cache miss" feels off to me. Seems like unpackaging of the file would only happen, in this example, on "cache hit" (i.e. the tar exists locally or within GCS).
```
FILE="$1"
CMD="$2"

if [[ ! -f ${FILE} && ! $UPLOAD_BIN cp ${PREFIX}/${FILE} . ]]; then
  set -e
  echo "*** Cache miss -- executing preprocessing-processing steps ***"
  bash -c "$CMD"
  $UPLOAD_BIN cp ${FILE} ${PREFIX}/${FILE}
else
  set -e
  echo "*** Cache Hit -- executing post-processing step ***"
  # input arg 3 represents optional `post-processing` step
  [[ -n $3 ]] && bash -c "${3}"
fi
```